odxtools/text.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from xml.etree import ElementTree
+
+from .odxlink import OdxDocFragment
+
+
+@dataclass
+class Text:
+    text: str
+    text_identifier: Optional[str]
```

Comment:
```suggestion
   identifier: Optional[str]
```

Reply:
I don't mind changing this in principle, but the XML attribute is called "TI" and the spec says that it stands for "text identifier" (same for DESCRIPTION tags, BTW)...